### PR TITLE
:persevere: add db info to template

### PR DESCRIPTION
**Pull request by beseven**
Add mongo db image to stack

### CODE REVIEW

File: templates/c2s-gp-finder/0/rancher-compose.yml [hunk 1]
```diff
@@ -11,6 +11,12 @@ catalog:
       required: true
       default: "latest"
       type: "string"
+    - variable: "gp_finder_db_docker_image_tag"
```
Comment (st3v3nhunt) [inline templates/c2s-gp-finder/0/rancher-compose.yml]: The variable will need to be `profiles_db_docker_image_tag` for it to work with the CI scripts. @neilbmclaughlin can you confirm?
If it does, probably the description and label here will want to be updated along with where it is used in `docker-compose` i.e. https://github.com/nhsuk/nhsuk-rancher-templates/pull/28/files#diff-a5f45e5d628f0630d457041224a3c35eR23

**Review by st3v3nhunt — CHANGES_REQUESTED** [review]
I think the variable name needs updating

File: templates/c2s-gp-finder/0/docker-compose.yml [hunk 2]
```diff
@@ -16,4 +16,13 @@ services:
...
+    links:
+      - gp-finder-db:mongo
 
+  gp-finder-db:
+    image: "nhsuk/profiles-db:${gp_finder_db_docker_image_tag}"
+    labels:
+      traefik.enable: true
+      traefik.domain: ${traefik_domain}
+      traefik.port: 27017
+      io.rancher.container.pull_image: always
```
Comment (neilbmclaughlin) [inline templates/c2s-gp-finder/0/docker-compose.yml]: New line.

File: templates/c2s-gp-finder/0/rancher-compose.yml (hunk 1 above)
Comment (neilbmclaughlin) [reply templates/c2s-gp-finder/0/rancher-compose.yml]: @st3v3nhunt is right. If you are using the profiles db then the variable will need to be called `profiles_db_docker_image_tag` for the CI scripts to work. 

If we change to having a _gp-finder-db_ repo with the Dockerfile extending the profiles db image to add the geospatial index we will use the `gp_finder_db_docker_image_tag` form. 

See the readme in [PR4](https://github.com/nhsuk/ci-deployment/pull/4) for more details on the naming conventions we currently employ.

**Review by neilbmclaughlin — CHANGES_REQUESTED** [review]

**Review by st3v3nhunt — APPROVED** [review]
Approved against [4574c89](https://github.com/nhsuk/nhsuk-rancher-templates/pull/28/commits/4574c8919ad32e231e899a1c07e2e21390e4ce78)

**Comment by neilbmclaughlin** [comment]
Approved against 4574c89

**Review by neilbmclaughlin — APPROVED** [review]

**Pull request merged**